crates/signal/src/bootstrap.rs

```diff
@@ -43,6 +40,11 @@ pub trait SignalConfig: Global {
 
         std::future::ready(Err(err))
     }
+
+    /// The global shutdown future.
+    fn block_global_shutdown(&self) -> impl std::future::Future<Output = ()> + Send {
+        scuffle_context::Handler::global().shutdown()
+    }
```

Comment:
Maybe add some more documentation  as to what this does exactly.

So in this case this function allows the user to override the context that is shutdown when the signal is canceled.